plugin/showpass-wordpress-plugin-shortcode.php

```diff
@@ -281,31 +285,12 @@ function showpass_get_timezone_abbr ($timezone) {
 	}
 }
 
-/* RETURN TRUE IF ALL TICKET TYPES ARE SOLD OUT */
-function showpass_ticket_sold_out ($data) {
```

Comment:
@marc-thomas So I changed this because of recurring events - the function name no longer made sense. So that would break all the templates that are using that function once that gets merged in and the user updates their plugin right? Wasn't sure we just let the user deal with it when they update the plugin or try to accommodate for less breakage?

Reply:
hey @april-duong, we shouldn't update function names as it can break custom templates if people update the plugin - please revert and use `showpass_ticket_sold_out` as the function name - you can update the comment to reflect the new logic of the function.